catalog/transformer/catalog-transformer-zip/src/test/java/org/codice/ddf/catalog/transformer/zip/TestZipCompression.java

```diff
@@ -90,83 +116,139 @@ public void setUp() throws Exception {
         when(resource.getName()).thenReturn(LOCAL_RESOURCE_FILENAME);
         when(resource.getInputStream()).thenReturn(resourceFileStream);
         ResourceResponse resourceResponse = new ResourceResponseImpl(resource);
-        when(catalogFramework.getLocalResource(any(ResourceRequest.class))).thenReturn(
+        when(catalogFramework.getLocalResource(any(ResourceRequestById.class))).thenReturn(
                 resourceResponse);
         zipCompression.setCatalogFramework(catalogFramework);
+    }
 
+    @After
+    public void tearDown() throws Exception {
+        Files.deleteIfExists(file.toPath());
```

Comment:
Would a [TemporaryFolder](http://junit.org/junit4/javadoc/latest/org/junit/rules/TemporaryFolder.html) work here? I think it would if you changed your zip file path to something like `tempFolder.getRoot().getAbsolutePath() + File.separator + "signed.zip"`. Might be worth looking into if it makes your code simpler.